dev-server/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "env": {
+    "node": true,
+  },
+  "parserOptions": {
+    "sourceType": "script"
```

Comment:
In the ESLint config, the parser was changed from the default to a TypeScript-aware one. The TS-supporting parser assumes .js/.ts files are ES modules by default, and so we need to tell ESLint explicitly about the non-modules in the codebase. This is the opposite from the previous behavior.